packages/babel-traverse/test/fixtures/rename/parameter-default-11/input.js

```diff
@@ -0,0 +1,7 @@
+function test(b) {
```

Comment:
Can you rename the test case by `rename/function-body-declaration`? It seems to me not that related to `parameter-default`.

Reply:
done